evaluation/comparison_with_existing_platforms.md

```diff
@@ -51,6 +51,31 @@ Like the MTPP, IFTTT also pays attention to the availability of the services, ho
 First, IFTTT uses a status endpoint to check if the services are available which is polled periodically. If a service is unavailable too often IFTTT automatically deactivates the corresponding routines [[1]].
 IFTTT requires the providers of the API to return multiple last event so that in case one event was missed it can be handled later. Further it supports webhook call-back requests for instant triggering of events. It supports webhook call-back request and polling of multiple recent events for triggers[[4]].
 
+## Conclusion
+### Comparison
+After looking at different competing platforms we want to give a general resume on how the MTPP compares to these platforms.
+
+All platforms allow some kind of routine representation but allows different degress of complexity and different representation. SCI and NodeRed allow more complex routines which are displayed similar to a process model. MTPP and IFTTT only allow simple routines which are represented as a textual list. MTPP's and IFTTT's Routine model consist of only two simple types of components, condition and operation. The other two platforms have a larger of number of complex components. In the end SCI and NodeRed allow more complex routines but are also more complicated to define by users.
+
+All platforms come with predefined operations that integrate other webservices into the plattform. MTPP, NodeRed and SCI allow custom integration of new webservices which is not allow with IFTTT. In IFTTT providers of a webservices must request and prepare the integration to the platform but receive some support.
+MTPP is the only platform that allows the built-in definition of alternative APIs for operations so that unavailable APIs can be replaced.
+
+### Features List
```

Comment:
another important feature:
HIghly extensibility of data model (du to the storage of columns, datatypes in our DB instead of "hardcoding" them)

Reply:
That is an advantage but more a technical feature than a functional feature. I listed only functional features. I think if we list both, it is a bit messy. Also since we did not analyze the code of the other platforms.